### PR TITLE
fixed linter result bug

### PR DESCRIPTION
**Pull request by lorenzschreyer**

### CODE REVIEW

**Review by KRIYO144 — APPROVED** [review]

**Review by vis9494 — APPROVED** [review]

**Review by Marcodaum — CHANGES_REQUESTED** [review]
\\n was also set above. For our machine the \n had to be escaped -> \\n....

**Comment by lorenzschreyer** [comment]
@Marcodaum on my machine if "\\n" is used instead of "\n" the get Result method returns a empty list. Also this is probably the fix for the Problem of @vis9494, because she also gets an empty list.

File: backend/codingchallenge/api/include/procedureGithubAPI.py [hunk 1]
```diff
@@ -87,8 +87,10 @@ def get_linter_result(self, repo_name):
         # question mark
         cleanSummary = cleanSummary.replace(u"\u25EC", "?")
 
+        # print(cleanSummary)
```
Comment (FabianBretz) [inline backend/codingchallenge/api/include/procedureGithubAPI.py:90]: Remove debug prints

File: backend/codingchallenge/api/include/procedureGithubAPI.py [hunk 2]
```diff
@@ -97,6 +99,7 @@ def get_linter_result(self, repo_name):
                     if element:
                         row.append(element.strip())
                 result.append(row)
+        print(result)
```
Comment (FabianBretz) [inline backend/codingchallenge/api/include/procedureGithubAPI.py:102]: Same as above

**Review by FabianBretz — CHANGES_REQUESTED** [review]
Forgot to remove debug prints

File: backend/codingchallenge/api/include/procedureGithubAPI.py (hunk 2 above)
Comment (lorenzschreyer) [reply backend/codingchallenge/api/include/procedureGithubAPI.py]: done

File: backend/codingchallenge/api/include/procedureGithubAPI.py (hunk 1 above)
Comment (lorenzschreyer) [reply backend/codingchallenge/api/include/procedureGithubAPI.py]: done

**Review by Marcodaum — APPROVED** [review]

**Review by FabianBretz — APPROVED** [review]
lgtm

**Pull request merged**